### PR TITLE
removes gas_transfer_coefficent from clothing as it did nothing

### PR DESCRIPTION
**Pull request by ghost**
it literaly does nothing

### CODE REVIEW

**Comment by TemporalOroboros** [comment]
> it literaly does nothing

You want to know the best part? This has been here since the r4407 and as far as anyone can tell its never done anything.

File: code/modules/clothing/masks/_masks.dm [hunk 1]
```diff
@@ -64,8 +64,6 @@
 	mask_adjusted = !mask_adjusted
 	if(!mask_adjusted)
 		src.icon_state = initial(icon_state)
-		gas_transfer_coefficient = initial(gas_transfer_coefficient)
-		permeability_coefficient = initial(permeability_coefficient)
```
Comment (Couls) [inline code/modules/clothing/masks/_masks.dm:68]: accidental delete here?

Comment (AMonkeyThatCodes) [reply code/modules/clothing/masks/_masks.dm:68]: woops 😳

**Comment by LemonInTheDark** [comment]
The evil has been purged

**Comment by Ghilker** [comment]
#40940 close this

**Comment by timothymtorres** [comment]
Down with the shitcode.

**Comment by timothymtorres** [comment]
Close #19339 as well

File: code/modules/clothing/masks/_masks.dm [hunk 2]
```diff
@@ -66,17 +66,15 @@
...
 		clothing_flags |= visor_flags
+		permeability_coefficient = initial(permeability_coefficient)
```
Comment (Rohesie) [inline code/modules/clothing/masks/_masks.dm:70]:
```suggestion
```

Comment (Rohesie) [inline code/modules/clothing/masks/_masks.dm:70]:
```suggestion
		permeability_coefficient = initial(permeability_coefficient)
		clothing_flags |= visor_flags
```
Back to how it was, for lower diff.

File: code/__DEFINES/layers.dm [hunk 3]
```diff
@@ -10,15 +10,15 @@
 
 
 #define OPENSPACE_LAYER 600 //Openspace layer over all
-#define OPENSPACE_PLANE -9 //Openspace plane below all turfs
-#define OPENSPACE_BACKDROP_PLANE -8 //Black square just over openspace plane to guaranteed cover all in openspace turf
+#define OPENSPACE_PLANE -10 //Openspace plane below all turfs
+#define OPENSPACE_BACKDROP_PLANE -9 //Black square just over openspace plane to guaranteed cover all in openspace turf
```
Comment (Rohesie) [inline code/__DEFINES/layers.dm:14]: Are these changes intended?

File: code/_onclick/hud/plane_master.dm [hunk 4]
```diff
@@ -61,6 +61,49 @@
 	if(istype(mymob) && mymob.client && mymob.client.prefs && mymob.client.prefs.ambientocclusion)
 		add_filter("AO", 1, drop_shadow_filter(x = 0, y = -2, size = 4, color = "#04080FAA"))
 
+/atom/movable/screen/plane_master/item_outline
+	name = "item outline plane master"
+	plane = ITEM_OUTLINE_PLANE
+	appearance_flags = PLANE_MASTER //should use client color
+	blend_mode = BLEND_OVERLAY
```
Comment (Rohesie) [inline code/_onclick/hud/plane_master.dm:68]: Is this related, or an oversight?

File: code/game/objects/items.dm [hunk 5]
```diff
@@ -114,7 +114,6 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 	///What body parts are covered by the clothing when you wear it
 	var/body_parts_covered = 0
 	///Literally does nothing right now
```
Comment (LemonInTheDark) [inline code/game/objects/items.dm:116]: Nuke the comment too

**Review by Cyberboss — CHANGES_REQUESTED** [review]
^

**Comment by LemonInTheDark** [comment]
gahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

**Comment by Cyberboss** [comment]
> 
> 
> gahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

Did you disable your boo-womp inhibitors?

**Review by Cyberboss — APPROVED** [review]
